tools/uw-frame-components/build.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+var less = require('less');
+var exec = require('child_process').exec;
+var glob = require('glob');
+
+var exec_handler = function(error, stdout, stderr) {
+    if (error) throw error;
+}
+
+exec('mkdir -p target/uw-frame-components', exec_handler);
+exec('cp -r uw-frame-components/* target/uw-frame-components', exec_handler);
```

Comment:
@paulerickson agreed. It will take a little effort to ensure the java and static versions can work without much if any variation.